packages/ui/src/theme/__tests__/overrides.test.ts

```diff
@@ -1067,12 +1067,7 @@ describe('@aws-amplify/ui', () => {
         --amplify-components-storagemanager-previewer-body-padding-block: var(--amplify-space-medium);
         --amplify-components-storagemanager-previewer-body-padding-inline: var(--amplify-space-medium);
         --amplify-components-storagemanager-previewer-body-gap: var(--amplify-space-small);
-        --amplify-components-storagemanager-previewer-footer-border-color: var(--amplify-colors-border-secondary);
-        --amplify-components-storagemanager-previewer-footer-border-style: solid;
-        --amplify-components-storagemanager-previewer-footer-border-width: var(--amplify-border-widths-small);
-        --amplify-components-storagemanager-previewer-footer-padding-block: var(--amplify-space-medium);
-        --amplify-components-storagemanager-previewer-footer-padding-inline: var(--amplify-space-medium);
-        --amplify-components-storagemanager-previewer-footer-justify-content: space-between;
```

Comment:
These were not used in the first place, storage manager did not have a footer. With the current UX borders don't look good so didn't need them.